addons/l10n_ar/demo/account_customer_refund_demo.xml

```diff
@@ -2,7 +2,7 @@
 <odoo noupdate="True">
 
     <!-- Create draft refund for invoice 3 -->
-    <record id="demo_refund_invoice_3" model="account.move.reversal">
+    <record id="demo_refund_invoice_3" model="account.move.reversal" context="{'allowed_company_ids': [ref('company_ri')]}">
```

Comment:
@jjscarafia Why can not we use the company_id on the invoice immediately?

Reply:
To be fair, it would be better if `company_id` was computed based on `move_ids`..., same for all the fields in the `default_get` (other than `move_ids` ofc)

Reply:
I was mainly concerned about the account.move but also in the wizard of the account.move.reversal the default is calculated based on the move_ids.

Reply:
Yes. That is exactly what I'm saying: it should be a computed field instead of being computed in the `default_get`

Reply:
@william-andre @jco-odoo please let me know if I need to take care of anything.
thanks!